github/pulls_comments.go

```diff
@@ -156,6 +156,30 @@ func (s *PullRequestsService) CreateCommentInReplyTo(ctx context.Context, owner
 	return c, resp, nil
 }
 
+// CreateReviewCommentReply creates a reply to a review comment for a pull request.
+//
+// GitHub API docs: https://developer.github.com/v3/pulls/comments/#create-a-review-comment-reply
+func (s *PullRequestsService) CreateReviewCommentReply(ctx context.Context, owner string, repo string, prNumber int, body string, commentID int64) (*PullRequestComment, *Response, error) {
+	comment := &struct {
+		Body string `json:"body,omitempty"`
```

Comment:
You can remove the `,omitempty` here since `body` must be supplied (and it is not a pointer to a `string` anyway so the `omitempty` would have no effect).